Library/Homebrew/cmd/bottle.rb

```diff
@@ -192,10 +199,19 @@ def bottle_formula(f)
 
         keg.delete_pyc_files!
 
+        tab = Tab.for_keg(keg)
+        tab["HEAD"] = formula_last_commit_hash
+        tab["time"] = formula_last_commit_time.to_i
+        tab.write
```

Comment:
I think capturing the commit hash and commit time at this stage basically gets us nowhere. Due to our contribution model, both will have changed by the time the commit ends up in our `master` and the information in the tab inside the bottle won't help to find or verify the commit to work with.